lib/koans/12_tuples.ex

```diff
@@ -0,0 +1,43 @@
+defmodule Tuples do
+  use Koans
+
+  koan "tuples have a size" do
+    assert tuple_size({:a, :b, :c}) == :__
+  end
+
+  koan "tuples can contain different things" do
+    assert {:a, 1, "hi"} == :__
+  end
+
+  koan "you can pull out individual elements" do
+    assert elem({:a, "hi"}, 1) == :__
+  end
+
+  koan "you can change individual elements of a tuple" do
+    assert put_elem({:a, "hi"}, 1, "bye") == :__
+  end
+
+  koan "you can also simply extend a tuple with new stuff" do
+    assert Tuple.insert_at({:a, "hi"}, 1, :new_thing) == :__
+  end
+
+  koan "...avoid falling of the edge" do
+    assert_raise :__, fn ->  Tuple.insert_at({:a, "hi"}, 12, :new_thing) end
```

Comment:
Tough call.  Should we put a hint on the name or even drop it altogether?
As I think through it, this kind of negative-case is close to the useless-edge.
You learn naturally about exceptions as they happen :smile: